src/dev-app/BUILD.bazel

```diff
@@ -158,10 +160,36 @@ dev_server(
         "@npm//:node_modules/tslib/tslib.js",
         "@npm//:node_modules/zone.js/dist/zone.js",
     ],
+)
+
+dev_server(
+    name = "devserver",
+    srcs = [":dev_app_static_files"],
     additional_root_paths = [
         "npm/node_modules",
+        # Needed for compatibility with "pkg_web" which always uses the tree
+        # artifact output as workspace root.
+        "angular_material",
     ],
     deps = [
         ":dev-app",
     ],
 )
+
+devmode_js_sources(
```

Comment:
Add comments for this and for the `pkg_web` target?

Reply:
Added.